game/src/Game/GameSession/ProvinceOverviewPanel.gd

```diff
@@ -305,12 +307,11 @@ func _update_info() -> void:
 			_rgo_icon.set_icon_index(_province_info.get(_province_info_rgo_icon_key, -1) + 2)
 
 		if _rgo_produced_label:
-			# TODO - replace name with amount produced
-			_rgo_produced_label.text = _province_info.get(_province_info_rgo_name_key, _province_info_rgo_name_key + _missing_suffix)
+			_rgo_produced_label.text = GUINode.float_to_string_dp(_province_info.get(_province_info_rgo_output_quantity_yesterday_key, 0), 3)
 
 		if _rgo_income_label:
-			# TODO - add £ sign and replace placeholder with actual value
-			_rgo_income_label.text = "%s¤" % GUINode.float_to_string_dp(12.34567, 3)
+			# TODO - add £ sign (?)
```

Comment:
@Hop311 the UI already shows £, see https://discord.com/channels/1063392556160909312/1063416834650554398/1301319793831575593
What should be done here?

(Could be placed out of scope of this PR)

Reply:
I think I forgot to remove the comment when I added the £ sign